2016/ideas-list-swc.md

```diff
@@ -231,14 +232,100 @@ Other: please use our [mailing list][].
 Thanks to @xuf12 for the initial idea behind this project and for @wking for
 providing this idea for GSOC'15.
 
+---------
+
+## Manage workflow for instructor training
+
+**Please ask questions as issues [here][gsoc-issues].**
+
+### Abstract
+
+Extend [AMY][] to manage the process of training people to be instructors.
+
+| **Intensity** | **Involves**  | **Mentors** |
+| ------------- | --------------|------------ |
+| Moderate | Workflow Design, Interface Design, Python, Django, Testing | [@pbanaszkiewicz][] |
```

Comment:
I can mentor the technical part, but you, @gvwilson, should be here as a consulting mentor – you know the great deal of training process and you know your specific requirements.